### DISCUSSION TITLE
Bots leave diving suits at the station, and then if you save and exit, the suits are irretrievably lost.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13652] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Bots leave diving suits at the station, and then if you save and exit, the suits are irretrievably lost.

### Reproduction steps

If the bot was wearing a diving suit when docking, bot wears off the suit at the station immediately after dock. If you save the game after this, the suit is lost.
One day I lost all my suits at once.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Was ticketed and resolved in: https://github.com/FakeFishGames/Barotrauma/issues/13669
